src/main/java/frc/robot/subsystems/arm/ScoreHighState.java

```diff
@@ -44,22 +44,18 @@ public void init() {
             return;
         }
 
-        double modifier = RobotMap.arm.getScoringDirectionModifier();
```

Comment:
We do not know for sure that we will not be scoring out the back yet. A better approach would be to make the `getScoringDirectionModifier()` always return 1.0 for now.